src/core/StelLocationMgr.cpp

```diff
@@ -1569,18 +1569,19 @@ void StelLocationMgr::changePlanetMapForLocation(StelLocation loc)
 	if (loc.planetName==planetName)
 		return;
 
-	QString mapName;
-	if (loc.planetName=="Earth")
-		mapName="data/gui/miscWorldMap.jpg";
-	else
-		mapName="textures/" + loc.planetName + ".png";
 	planetName=loc.planetName;
-
-	if (!planetSurfaceMap.load(StelFileMgr::findFile(mapName, StelFileMgr::File)))
+	if (planetName=="Earth")
+		planetSurfaceMap=QImage(":/graphicGui/miscWorldMap.jpg");
+	else
 	{
-		// texture not found. Use a gray pixel.
-		planetSurfaceMap=QImage(16,16,QImage::Format_RGB32);
-		planetSurfaceMap.fill(QColor(64, 64, 64));
+		QString mapName="textures/" + planetName + ".png";
```

Comment:
`LocationDialog` searches for the map to pass to `MapWidget` as follows:
```
StelFileMgr::findFile("textures/"+p->getTextMapName())
```
Here `p = ssm->searchByEnglishName(loc.planetName)`. Maybe this should be done here the same way.

Reply:
Done.